dev/benchmarks/complex_layout/test/measure_scroll_smoothness.dart

```diff
@@ -167,7 +164,7 @@ Future<void> main() async {
         } else if (delays.last < delay) {
           delays.last = delay;
         }
-        tester.binding.handlePointerEventForSource(event, source: TestBindingEventSource.test);
+        tester.binding.handlePointerEvent(event, source: TestBindingEventSource.test);
```

Comment:
Hello @jmagman 
`handlePointerEventForSource` this method is not present now.
So I thought this could be the alternative.

Reply:
Why's the method not present? It appears to still exist here: https://master-api.flutter.dev/flutter/flutter_test/TestWidgetsFlutterBinding/handlePointerEventForSource.html